common/lib/capa/capa/templates/choicetext.html

```diff
@@ -25,6 +25,8 @@
                     correctness = 'correct'
                 elif status == 'incorrect':
                     correctness = 'incorrect'
+                elif status == 'partially-correct':
+                    correctness = 'partialcorrect'   #Yes, this is different from how it's named elsewhere. :(
```

Comment:
The only reference to "partialcorrect" I see in the codebase is in ORA1,
which is now deprecated. My vote is to change this to "partially-correct".
Is there another reference I'm missing?

On Tue, Jun 23, 2015 at 3:48 PM, Colin-Fredericks <notifications@github.com>
wrote:

> In common/lib/capa/capa/templates/choicetext.html
> <https://github.com/edx/edx-platform/pull/8574#discussion_r33083205>:
>
> > @@ -25,6 +25,8 @@
> >                      correctness = 'correct'
> >                  elif status == 'incorrect':
> >                      correctness = 'incorrect'
> > +                elif status == 'partially-correct':
> > +                    correctness = 'partialcorrect'   #Yes, this is different from how it's named elsewhere. :(
>
> Because the css class names are different from the correctness designation
> in the python, and I didn't want to change them in case someone was
> actually using them.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/8574/files#r33083205>.
>